capa/ghidra/capa_explorer.py

```diff
@@ -200,7 +200,11 @@ def label_matches(self):
                                 if sub_func is not None:
                                     sub_func_addr = sub_func.getEntryPoint()
                                     # place function in capa namespace & create the subscope match label in Ghidra's global namespace
-                                    create_label(sub_func_addr, sub_func.getName(), capa_namespace)
+                                    create_label(
+                                        sub_func_addr,
+                                        sub_func.getName(),
+                                        capa_namespace,
+                                    )
```

Comment:
Let's revert this to the previous formatting unless one of the linters is complaining.

Reply:
i think the linter threw an error and I committed the format , I can revert back to the old formatting it is more readable that way . If it still throws an error in the CI then I will just  keep it like above